purchase_freight_delay/models/purchase.py

```diff
@@ -9,42 +9,61 @@
 class PurchaseOrder(models.Model):
     _inherit = "purchase.order"
 
-    freight_rule_id = fields.Many2one(comodel_name="freight.rule")
-    freight_duration = fields.Integer(compute="_compute_freight", store=True)
-    already_computed_freight = fields.Boolean(readonly=True, help="Flag to ")
-    previous_date_planned = fields.Datetime(copy=False)
+    freight_rule_id = fields.Many2one(comodel_name="freight.rule", copy=False)
+    freight_duration = fields.Integer(
+        compute="_compute_freight_duration",
+        inverse="_inverse_freight_duration",
+        store=True,
+        copy=False,
+    )
+    initial_planned = fields.Datetime(copy=False)
```

Comment:
create a PO, add a POL, change date: it's reverted to initial_planned. Not what we expect